bin/clone

```diff
@@ -45,15 +44,24 @@ def main():
     else:
         log_file = "/opt/snare/clone.log"
     logger.Logger.create_clone_logger(log_file, __package__)
-    print_color("Logs will be stored in {}\n".format(log_file), 'INFO')
+    print("  Logs will be stored in {}".format(log_file), end="")
     try:
+        start=datetime.now()
         cloner = Cloner(args.target, int(args.max_depth), args.css_validate)
         loop.run_until_complete(cloner.get_root_host())
         loop.run_until_complete(cloner.run())
+        end=datetime.now()-start
+        print("")
+        print("""
+        -------------------------------------->SUMMARY<------------------------------------------------
+         Total number of URLs cloned: {}
+         Time elapsed: {}
+         Cloned directory: {}
+        -----------------------------------------------------------------------------------------------
+        """.format(str(cloner.counter), str(end), cloner.target_path))
     except KeyboardInterrupt:
```

Comment:
what is we want summary also if we interrupt execution?

Reply:
Agreed, I'll get it done :+1: